### ISSUE TITLE
install is broken

### ISSUE THREAD

**Issue by kanekv** [GoogleCloudPlatform/opentelemetry-operations-go#35]
```
go get -u github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/trace

../../../go/src/pkg/mod/github.com/!google!cloud!platform/opentelemetry-operations-go/exporter/trace@v0.1.0/trace_proto.go:30:2: module go.opentelemetry.io/otel@latest found (v0.6.0), but does not contain package go.opentelemetry.io/otel/api/core
../../../go/src/pkg/mod/github.com/!google!cloud!platform/opentelemetry-operations-go/exporter/trace@v0.1.0/cloudtrace.go:28:2: module go.opentelemetry.io/otel@latest found (v0.6.0), but does not contain package go.opentelemetry.io/otel/api/key
```

**Comment by kanekv** [comment]
Apparently it's fixed in master branch, but not published.

**Comment by nilebox** [comment]
@kanekv I have released [v0.2.0](https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/releases/tag/v0.2.0), please let me know if that resolved your issue:

**Comment by kanekv** [comment]
This works, thanks!
